truss/util/env_vars.py

```diff
@@ -12,21 +12,32 @@ class override_env_vars:
         # Original environment is restored here
     """
 
-    def __init__(self, env_vars: Dict[str, str]):
+    def __init__(
```

Comment:
(optional): Now that we support deletions, I wonder if `modify_env_vars` makes more sense as this class name, and then the params could be `overrides` and `deletions`